docker-compose.yml

```diff
@@ -32,3 +32,5 @@ services:
 
   avro_tools: # https://hub.docker.com/r/coderfi/avro-tools
     image: coderfi/avro-tools:1.7.7
+    ports:
+     - '9999:9999'
```

Comment:
Why do we need to expose this port for avro-tools?

Reply:
There is a bug in dockest I believe where the docker-compose must define a port.  I'll look into the root cause and see if there is a workaround that doesn't require this hack.

Reply:
@erikengervall is this something I should address in dockest, or is there a known workaround?

Reply:
> @erikengervall is this something I should address in dockest, or is there a known workaround?

Hey! There's even a version 3.0.0 of Dockest in the making which will include a number of improvements. Can't tell exactly what the ETA is, but there's a few PRs queued to be merged and some beta testing required before we can release.

> I'll look into the root cause and see if there is a workaround that doesn't require this hack.

I'd say this port-hack is okay for now, I could circle back to this piece of code once Dockest v3 is out and this has been addressed 🙂